### DISCUSSION TITLE
have different level  (options ?) of flexibilities

### DISCUSSION THREAD

**Discussion by magnetik** [CuyZ/Valinor#168] [Ideas] [answered]
hi,

I'm trying to follow up on the quick updates you do, and I've just read about the flexible option.

While I prefer to be flexible on the "non mapped values"  , I do not mind being strict on the types.

Can different configuration exist for flexibilities? 

It might be overkill, just my two cents!

Thanks

**Comment by romm** [comment]
Hi @magnetik, unless you (or others) have a real reason to be willing to separate these two features, there is no plan to handle several configurations for it.

Thanks for the suggestion, though. 😉

**Comment by romm** [comment, marked as answer]
Hi @magnetik, just to let you know that in the end this feature has been challenged by #203 and #235 and has been proposed with #248.

If you have some time to check it out that could help. 🙂

**Reply by magnetik** [reply]
That looks perfect ! Thanks :)
